msw/msw-app.sh

```diff
@@ -206,8 +206,16 @@ if [ "x$prototype_tk" = xfalse ] ; then
     cp -R $TK/bin $APP/
     cp -R $TK/lib $APP/
 
-    # remove some Tcl libraries Pd doesn't need
-    rm -rf $APP/lib/sqlite* $APP/lib/tdbc*
+    # remove bundled Tcl packges Pd doesn't need
+    rm -rf $APP/lib/itcl* $APP/lib/sqlite* $APP/lib/tdbc*
+
+    # install pthread from MinGW
+    if [ -v MINGW_PREFIX ] ; then
```

Comment:
hmm, so compiling with `MINGW_PREFIX=""` will evaluate to *true*. i think a simple check for non-emptiness is probably more what we want.

Reply:
Fixed.